react/training/src/app/screens/Login/components/index.js

```diff
@@ -0,0 +1,9 @@
+import React, { Fragment } from 'react';
```

Comment:
This should be in `components/CustomInput/index.js`. Create a folder with the component name. Otherwise what are you going to do when you have two components like `Text` or `Button` ?